src/main/java/io/ticktok/client/server/rest/RestClockActions.java

```diff
@@ -38,8 +42,14 @@ private Clock getClockBy(ClockRequest clockRequest) throws IOException {
         final HttpGet httpGet = new HttpGet(urlResolver.queryParam("name", clockRequest.getName()).queryParam("schedule", clockRequest.getSchedule()).resolve());
         HttpResponse httpResponse = httpClient.execute(httpGet);
         String entity = extractEntityFrom(httpResponse);
-        new RestResponseValidator(httpResponse).validate(200, new ClockNotFoundException(entity));
-        return clockFrom(entity);
+        final List<Clock> clocks = new Gson().fromJson(entity, new TypeToken<List<Clock>>() {}.getType());
+        if(clocks.isEmpty()) {
+            throw new ClockNotFoundException(format(
+                    "Clock named: %s, schedule: %s isn't currently configured",
+                    clockRequest.getName(),
+                    clockRequest.getSchedule()));
+        }
+        return clocks.get(0);
```

Comment:
i would extract the 0 to a const with meaningful name

Reply:
some reaction to this offer :)...

Reply:
I don't think 0 will confuse a programmer here, and the function name clearly state the intent